backend/database/organization.py

```diff
@@ -0,0 +1,19 @@
+from config import db
+
+class Organization(db.Model):
+    __tablename__ = 'organization'

```

Comment:
Please make the name to be capitalize 'Organization' so that everyone is on the same naming.